### PR TITLE
{geo}[foss/2020b] pysheds v0.2.7.1, pyproj v3.0.1, rasterio v1.2.3 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/rasterio/rasterio-1.2.3-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+use_pip = True
+
+exts_list = [
+    ('affine', '2.3.0', {
+        'checksums': ['2e045def1aa29e613c42e801a7e10e0b9bacfed1a7c6af0cadf8843530a15102'],
+    }),
+    ('attrs', '20.3.0', {
+        'modulename': 'attr',
+        'checksums': ['832aa3cde19744e49938b91fea06d69ecb9e649c93ba974535d08ad92164f700'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/r/rasterio/rasterio-1.2.3-foss-2020b.eb:27]: Python 3.8.6 provides `attrs` 20.2.0 (https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-3.8.6-GCCcore-10.2.0.eb#L226) - do we need the slightly newer version here?

File: easybuild/easyconfigs/r/rasterio/rasterio-1.2.3-foss-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    ('s3transfer', '0.4.2', {
+        'checksums': ['cb022f4b16551edebbb31a377d3f09600dbada7363d8c5db7976e7f47732e1b2'],
+    }),
+    ('boto3', '1.17.61', {
+        'checksums': ['35b099fa55f5db6e99a92855b9f320736121ae985340adfc73bc46fb443809e9'],
+    }),
+    ('click', '7.1.2', {
+        'checksums': ['d2b5255c7c6349bc1bd1e59e08cd12acbbd63ce649f2588755783aa94dfb6b1a'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/r/rasterio/rasterio-1.2.3-foss-2020b.eb:42]: Python 3.8.6 provides this version of `click` (https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-3.8.6-GCCcore-10.2.0.eb#L303) - so this should be removed.

**Review by branfosj — DISMISSED** [review]

File: easybuild/easyconfigs/r/rasterio/rasterio-1.2.3-foss-2020b.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/r/rasterio/rasterio-1.2.3-foss-2020b.eb]: indeed, it's not necessary

File: easybuild/easyconfigs/r/rasterio/rasterio-1.2.3-foss-2020b.eb (hunk 2 above)
Comment (smoors) [reply easybuild/easyconfigs/r/rasterio/rasterio-1.2.3-foss-2020b.eb]: fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/12791/commits/840bd55e71099051b1e492e176243dca56ca27bb

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/669f603222cc4f2958b704b237a26c24 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @smoors!

**Pull request merged**